weis/aeroelasticse/openmdao_openfast.py

```diff
@@ -1403,6 +1426,8 @@ def update_FAST_model(self, fst_vt, inputs, discrete_inputs):
             fst_vt['HydroDyn']['MJointID2'] = fst_vt['HydroDyn']['MPropSetID2'] = N2
             fst_vt['HydroDyn']['MDivSize'] = 0.5*np.ones( fst_vt['HydroDyn']['NMembers'] )
             fst_vt['HydroDyn']['MCoefMod'] = np.ones( fst_vt['HydroDyn']['NMembers'], dtype=np.int_)
+            fst_vt['HydroDyn']['MHstLMod'] = np.ones( fst_vt['HydroDyn']['NMembers'], dtype=np.int_)
+            fst_vt['HydroDyn']['PropPot'] = np.ones( fst_vt['HydroDyn']['NMembers'], dtype=np.int_)
```

Comment:
I think this should be `fst_vt['HydroDyn']['PropPot'] = [False] * fst_vt['HydroDyn']['NMembers']` by default for strip-theory only models. Similarly, we might change this to be model-dependent or user-specified.

Reply:
I agree with @yqliaohk, `PropPot` should be a boolean flag. I also think that this would ideally be a user-specified value. In case it is too cumbersome to define the value for each member, maybe a single flag specifying whether the whole model is strip-theory only, potential-flow only, or hybrid models would be useful

Reply:
Nevermind, just saw that there's a flag "potential_model_override" later. I cannot add comments to lines that weren't modified, but do we need the code in https://github.com/WISDEM/WEIS/blob/73e56c53038e28bb3c649d8b481cc2c4e84d1a48/weis/aeroelasticse/openmdao_openfast.py#L1440 now that `PropPot` is defined beforehand?

Reply:
Agreed.  

`PropPot` should be defined for all cases below, based on `potential_model_override`